components/hidden-text/package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.2.5",
   "dependencies": {
     "@govuk-react/constants": "^0.2.2",
-    "@govuk-react/inset-text": "^0.2.5",
-    "@govuk-react/paragraph": "^0.2.5",
+    "@govuk-react/hoc": "^0.2.4",
```

Comment:
probably need to rebase the branch off master as it seems to regress the version numbers here?

Reply:
Really odd - wonder if this is project wide ~ I might have mashed something in the release. Thanks for the update.